js/bitget.js

```diff
@@ -2387,66 +2379,111 @@ module.exports = class bitget extends Exchange {
         }
         const response = await this[method] (this.extend (request, query));
```

Comment:
@Dan-krm It seems there are endpoints for stop order history: https://bitgetlimited.github.io/apidoc/en/spot/#get-history-plan-orders and https://bitgetlimited.github.io/apidoc/en/mix/#get-history-plan-orders-tpsl

Should we check whether it's stop or not?